proto/dymensionxyz/dymension/forward/events.proto

```diff
@@ -5,9 +5,12 @@ option go_package = "github.com/dymensionxyz/dymension/v3/x/forward/types";
 
 
 message EventForward{
+    // success?
     bool ok = 1;
     // empty if ok is true
     string err = 2;
+    // was it actually a forward operation? (maybe not if they dont include forward memo)
+    bool was_forwarded = 3;
```

Comment:
whats the use of this was_forwarded? seems like this event simply shouldn't be emitted if it's not forwarded no?